doc/modules/pipeline.rst

```diff
@@ -138,15 +138,18 @@ transfomers or a `joblib.Memory<https://pythonhosted.org/joblib/memory.html>`_
 object::
 
     >>> from tempfile import mkdtemp
+    >>> from shutil import rmtree
     >>> from sklearn.decomposition import PCA
     >>> from sklearn.svm import SVC
     >>> from sklearn.pipeline import Pipeline
-    >>> estimators = [('reduce_dim', PCA()), ('clf', SVC())]
-    >>> pipe = Pipeline(estimators, memory=mkdtemp())
+    >>> param_grid = [('reduce_dim', PCA()), ('clf', SVC())]
+    >>> cachedir = mkdtemp()
+    >>> pipe = Pipeline(param_grid, memory=cachedir)
     >>> pipe # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
     Pipeline(...,
              steps=[('reduce_dim', PCA(copy=True,...)),
-                    ('clf', SVC(C=1.0,...))])
+                   ('clf', SVC(C=1.0,...))])
+    >>> rmtree(cachedir)
```

Comment:
Maybe I would add here a comment "don't forget to clear the cache directory when you don't need it anymore.